frontend/tailwind.config.js

```diff
@@ -49,6 +49,10 @@ module.exports = {
         'accordion-down': 'accordion-down 0.2s ease-out',
         'accordion-up': 'accordion-up 0.2s ease-out',
       },
+      borders: {
+        test: 'border-right-style: dashed',
+        r: { test: 'border-right-style: dashed' },
+      },
```

Comment:
Are you using these?

Reply:
Uff it looks like I have pushed my test branch, not the correct one. I'll take a look.

Reply:
Test config removed.